src/status.md

```diff
@@ -0,0 +1,313 @@
+---
+description: Are we there yet?
+sidebarDepth: 2
+---
+
+# Status
+
+Not all language features are equally viable to implement on top of WebAssembly's current capabilities, and while AssemblyScript is already useful today, there is also still a lot to do. Keep in mind that WebAssembly is an evolving technology, and so is AssemblyScript.
+
+## Philosophy
+
+The original idea making AssemblyScript attractive is that it wants to be a thin and efficient layer on top of WebAssembly with a familiar syntax, ultimately producing lean and mean Wasm binaries. This idea is composed of two components, however, that are sometimes antagonal: **Lean and mean** implies that we have to stay close to WebAssembly, favoring future WebAssembly features over filler implementations in linear memory, while a **familiar syntax** naturally makes us want to support more of the original language right now. One could say that AssemblyScript is moving on a slippery slope by design, with the ultimate goal being to get rid of as much of the slope as we can without sacrificing efficiency.
+
```

Comment:
```suggestion

In other words, AssemblyScript wants to stay as close as possible to being a language that feels very familiar with JavaScript and TypeScript developers. AssemblyScript will also try it's best to preserve compatibility with the TypeScript compiler and tooling. However, AssemblyScript is a language focused on generating WebAssembly modules. AssemblyScript will prioritize language features that allow developers to write fast and small WebAssembly modules on top of the features that WebAssembly provides in their specification.
```

Reply:
Let me know if that stance sounds correct per our discussions in the working / community group. 

I tried to write this in a way that if I had a vague idea of what JavaScript and WebAssembly were, what is this language trying to do? :smile:

Reply:
I mangled your paragraph and mine a bit. Let me know if that's better respectively preserves your original suggestion :)

Reply:
Sounds good! Thanks dude! :smile: :tada: